src/app/modules/core/services/validator.service.ts

```diff
@@ -33,18 +33,27 @@ export class ValidatorService {
     ),
   );
 
-  performance$: Observable<ValidatorPerformanceResponse> = zip(
+  performance$: Observable<ValidatorPerformanceResponse & ValidatorBalances> = zip(
     this.beaconNodeService.nodeEndpoint$,
     this.walletService.validatingPublicKeys$
   ).pipe(
     switchMap((result: [string, string[]]) => {
-      const endpoint = result[0];
       const publicKeys = result[1];
+      const endpoint = result[0];
       let params = `?publicKeys=`;
       publicKeys.forEach((key, _) => {
         params += `${this.encodePublicKey(key)}&publicKeys=`;
       });
-      return this.http.get<ValidatorPerformanceResponse>(`${endpoint}/validators/performance${params}`);
+      const balances = this.balances(publicKeys, 0, publicKeys.length);
+      const httpReq = this.http.get<ValidatorPerformanceResponse>(`${endpoint}/validators/performance${params}`);
+      return zip(httpReq, balances).pipe(
```

Comment:
We now include the balances in the performance response to be able to easily compute total balance